pull_request_template.md

```diff
@@ -19,9 +19,8 @@ If you have any questions or problems, please give a comment in the PR.
 ##### General
 - [ ] PR name is meaningful
 - [ ] Purpose of the PR is understood
-- [ ] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
```

Comment:
I would like to keep one line for the CHANGELOG.md checks and not split it up into 4 single points.

Reply:
I changed it back to a single line and added a section in the Developer Guide.
I also changed "PR number referenced is same" into "PR number is verified.". I considered the initial wording misleading, however I'm also not to happy with my replacement. Maybe you got a better wording?

Reply:
I propose
```diff
+- [ ] A separate commit in CHANGELOG.md with this PR number exist and is placed on top of the section
```